font-patcher

```diff
@@ -700,6 +700,7 @@ class font_patcher:
             'firasans'       : 'furasans',
             'IntelOneMono'   : 'IntoneMono',
             'IntelOne Mono'  : 'Intone Mono',
+            'Intel One Mono' : 'Intone Mono',
```

Comment:
this might not be needed actually, the release version 3.1.1 does rename it properly

Reply:
Well, the legathy renaming in the `font-patcher` is not used, normally (probably we should remove it completely).
And the rule added by 
* d3181df01f8b438b52cefe212019f85fbd4d2c7e `name-parser: Add IntelOne replacement to SIL table`

is rather forgiving ;-)

(But this change is the correct thing to do, so keep it.)

Lets compare the actual names (via [1])

```
======== intelone-mono-font-family-regular.ttf ========
SFNT Fullname      ID 4     IntelOne Mono
SFNT Family        ID 1     IntelOne Mono
SFNT SubFamily     ID 2     Regular
SFNT Pref Family   ID 16    -
SFNT Pref Styles   ID 17    -
SFNT PS Name       ID 6     IntelOneMono-Regular
SFNT Compatible    ID 18    -
SFNT CID findfont  ID 20    -
SFNT WWS Family    ID 21    -
SFNT WWS SubFamily ID 22    -
PS fontname                 IntelOneMono-Regular
PS fullname                 IntelOne Mono
PS familyname               IntelOne Mono
fondname                    None

======== IntelOneMono-Regular.ttf ========
SFNT Fullname      ID 4     Intel One Mono
SFNT Family        ID 1     Intel One Mono
SFNT SubFamily     ID 2     Regular
SFNT Pref Family   ID 16    -
SFNT Pref Styles   ID 17    -
SFNT PS Name       ID 6     IntelOneMono-Regular
SFNT Compatible    ID 18    -
SFNT CID findfont  ID 20    -
SFNT WWS Family    ID 21    -
SFNT WWS SubFamily ID 22    -
PS fontname                 IntelOneMono-Regular
PS fullname                 Intel One Mono
PS familyname               Intel One Mono
fondname                    None
```

[1] `bin/scripts/name_parser/query_name`

Reply:
and ...

```
'firasans'       : 'furasans',
```

is technically wrong, which is good, because it is also wrong from the contents, so a bug fixes the bug ;-D

(I believe, the legacy renaming I never test)